internal/pkg/cli/local_run.go

```diff
@@ -297,16 +301,52 @@ func (o *localRunOpts) Execute() error {
 	}
 	o.imageInfoList = append(o.imageInfoList, sidecarImageLocations...)
 
-	err = o.runPauseContainer(context.Background(), ports)
-	if err != nil {
-		return fmt.Errorf("run pause container: %w", err)
-	}
+	g, ctx := errgroup.WithContext(context.Background())
+	gotSigInt := &atomic.Bool{}
+	finishedErr := errors.New("containers stopped")
 
-	err = o.runContainers(context.Background(), o.imageInfoList, envVars)
-	if err != nil {
+	g.Go(func() (err error) {
```

Comment:
nit
```suggestion
	g.Go(func() error {
```

Reply:
actually this is necessary in this case! you can see on line 313 i set the value of the error getting returned by the function to `nil` if we've already received a `SIGINT`/`SIGTERM`. This is bc as we are _expecting_ those functions to return an error at that point (since killing a container will cause `Run()` to exit), but we don't want to show those expected errors to the user.